docs/release-workflow-guide.md

```
@@ -147,6 +147,34 @@ On the **code freeze date (Friday)**, each team must:
 
 > **Note**: The comment format must comply with the expected structure to be parsed by the release automation tool.
 
+#### Operator-Level Images
+
+For operator-level images (non-component images like `kube-auth-proxy`), teams can include an additional section in their tracker comment:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)